src/io/json.jl

```diff
@@ -52,13 +53,45 @@ function parse_json(io::Union{IO,String}; kwargs...)::Dict{String,Any}
 
     _jsonver2juliaver!(pm_data)
 
-    if haskey(pm_data, "conductors")
-        _parse_mcv!(pm_data)
-    end
+    _parse_mats!(pm_data)
 
     if get(kwargs, :validate, true)
         PowerModels.correct_network_data!(pm_data)
     end
 
     return pm_data
 end
+
+
+function print_file(path::String, pmd_data)
+    open(path, "w") do io
+        print_file(io, pmd_data)
+    end
+end
+
+
+function print_file(io::IO, pmd_data)
+    JSON.print(io, JSON.parse(sprint(show_json, PMDSerialization(), pmd_data)))
+end
+
+
+function show_json(io::StructuralContext, ::PMDSerialization, f::Matrix{<:Any})
+    N, M = size(f)
+    value = string("[", join([join([f[i,j] for j in 1:M], " ") for i in 1:N], "; "), "]")
+    eltyp = isempty(f) ? eltype(f) : typeof(f[1,1])
+        out = Dict(:type=>:Matrix, :eltype=>eltyp, :value=>value)
+    return show_json(io, StandardSerialization(), out)
+end
```

Comment:
Just want to confirm I'm reading this right, matrices turn into `[a b c; d e f; g h i]` in the json string?

Reply:
Correct. I couldn't simply print the matrices because then they are not printed with full precision, leading to rounding of the numbers when you saved them.